### DISCUSSION TITLE
Species Seen but not Added to the Detected list

### DISCUSSION THREAD

**Discussion by dalogue1** [Nachtzuster/BirdNET-Pi#106] [General]
I have a problem with a BirdNET-Pi not logging a bird which it sees but will not add to my detected list.

I have just set up a BirdNET-Pi and it is identifying birds. It has identified many birds but not the Scaly Breasted Munias that are  all over my bird feeder and back yard.  I live in Houston, Texas.

It has identified the following but not the Scaly Breasted Munias

- American Robin
- Blue Jay
- Carolina Wren
- House Sparrow
- Northern Cardinal
- Northern Mockingbird
- White-winged Dove

This is very frustrating as I have about 10 of them on my feeder right now! I unchecked the Species range model and later also decreased the Species Occurance Frequency Threshhold from 0.03 to 0.003. Still no luck!

Any suggestions on how to get it to identify these birds?  They are easily identified in the Merlin Bird ID iPhone app from CornellLab.

Here is a sample of the log:

00:11:16---[birdnet_analysis][INFO] Analyzing BirdSongs/StreamData/2024-06-16-birdnet-00:11:01.wav
00:11:16---[server][INFO] READING AUDIO DATA...
00:11:16---[server][INFO] READING DONE! READ 5 CHUNKS.
00:11:16---[server][INFO] ANALYZING AUDIO...
00:11:17---[server][INFO] DONE! Time 0.51 SECONDS
00:11:17---[server][INFO] 0.0;3.0-('Lonchura punctulata_Scaly-breasted Munia', 0.45464423)
00:11:17---[server][INFO] 3.0;6.0-('Lonchura punctulata_Scaly-breasted Munia', 0.56192774)
00:11:17---[server][INFO] 6.0;9.0-('Lonchura punctulata_Scaly-breasted Munia', 0.61161417)
00:11:17---[server][INFO] 9.0;12.0-('Lonchura punctulata_Scaly-breasted Munia', 0.8099816)
00:11:17---[server][INFO] 12.0;15.0-('Lonchura punctulata_Scaly-breasted Munia', 0.3657702)
00:11:31---[birdnet_analysis][INFO] Analyzing BirdSongs/StreamData/2024-06-16-birdnet-00:11:16.wav
00:11:31---[server][INFO] READING AUDIO DATA...
00:11:31---[server][INFO] READING DONE! READ 5 CHUNKS.
00:11:31---[server][INFO] ANALYZING AUDIO...
00:11:32---[server][INFO] DONE! Time 0.51 SECONDS
00:11:32---[server][INFO] 0.0;3.0-("Accipiter cooperii_Cooper's Hawk", 0.15315849)
00:11:32---[server][INFO] 3.0;6.0-('Lonchura punctulata_Scaly-breasted Munia', 0.8867455)
00:11:32---[server][INFO] 6.0;9.0-('Human_Human', 0.0)
00:11:32---[server][INFO] 9.0;12.0-('Ramphocelus passerinii_Scarlet-rumped Tanager', 0.17540058)
00:11:32---[server][INFO] 12.0;15.0-('Human_Human', 0.0)

**Comment by dalogue1** [comment]
Here is a sample of the Merlin Bird ID app identification of the Scaly Breated Munias.
![IMG_3005](https://github.com/Nachtzuster/BirdNET-Pi/assets/133064876/e0c8801a-6387-4985-a624-683b40044a04)

**Comment by dalogue1** [comment]
The line in the Species list is as follows for my location:

Lonchura punctulata_Scaly-breasted Munia - 0.00055277866

After I lowered the Threshhold to 0.0005 it added the Scaly Breasted Munias

The birds identified are now:

- American Robin
- Black-backed Tanager
- Blue Jay
- Carolina Wren
- House Sparrow
- Northern Cardinal
- Northern Mockingbird
- Scaly-breasted Munia
- White-winged Dove
- Yellow-bellied Flycatcher

**Comment by dalogue1** [comment]
The Scaly Breasted Munia is not supposed to be in America at all.


![image](https://github.com/Nachtzuster/BirdNET-Pi/assets/133064876/a10768b5-7271-49b6-b85d-b9c205b9cada)

**Reply by ThomDyson** [reply]
Introduced species, with some monitoring going on. Houston Audubon wants to hear from you. https://houstonaudubon.org/birding/gallery/scaly-breasted-munia.html

**Reply by dalogue1** [reply]
I have about 20 that nest in the tree in my back yard.  I see them every day.
[Scaly-breasted_Munia.wav.zip](https://github.com/user-attachments/files/15964729/Scaly-breasted_Munia.wav.zip)
